src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -513,13 +513,25 @@ public function column_source( $post ) {
 		}
 
 		$html[] = '<p>' . esc_html_x( 'via ', 'record via origin', 'the-events-calendar' ) . '<strong>' . $source_info['via']  . '</strong></p>';
+
+		$html[] = '<div class="tribe-view-links-container" style="display:flex; flex-direction:row">';
+        if ( 'scheduled' === $this->tab->get_slug() ) {
+            $filter_link = admin_url( "edit.php?post_type=tribe_events&aggregator_record={$record->id}" );
+            $html[]      = '<div class="tribe-view-events-container">';
+            $html[]      = '<a href="' . esc_url( $filter_link ) . '" class="tribe-view-events">' . esc_html__( 'View Events', 'the-events-calendar' ) . '</a>';
+            $html[]      = '</div>';
+        }
+
 		if (
 			! empty( $record->meta['keywords'] )
 			|| ! empty( $record->meta['start'] )
 			|| ! empty( $record->meta['location'] )
 			|| ! empty( $record->meta['radius'] )
 		) {
 			$html[] = '<div class="tribe-view-filters-container">';
+            if ( 'scheduled' === $this->tab->get_slug() ) {
+                $html[] = '&nbsp;|&nbsp;';
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: ddabd145f8ef2f0428f3fa0a7ac3d761</sub>